cmake/FindSUNDIALS.cmake

```diff
@@ -31,10 +31,6 @@
 
 include(FindPackageHandleStandardArgs)
 
-find_package(SUNDIALS CONFIG QUIET)
```

Comment:
Is this failing in some case?

Reply:
Yes, I think this is for https://github.com/boutproject/BOUT-dev/issues/2381

I think I closed it because I had included this fix in my branch, thus thought it was fixed.